src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -300,12 +300,17 @@ const AppWrapper = (props: { children: ReactNode }) => {
       if (/SignifyClient/gi.test(errorStack)) {
         await loadDatabase();
         Agent.agent.bootAndConnect().then(() => {
+          setIsReady(Agent.agent.getKeriaOnlineStatus());
           setIsOnline(Agent.agent.getKeriaOnlineStatus());
         });
       } else {
         throw e;
       }
     }
+    Agent.agent.onKeriaStatusStateChanged((event) => {
```

Comment:
Maybe we can set this handler up earlier (before the try catch above), and not need the other calls to `setIsOnline` in this file. So it just gets updated with the handler only. This would be cleaner I think

Reply:
The `eventService` is initialized inside `start` so we need to run `start` before this line, else it will throw an error because the `eventService` is `undefined`.

Reply:
Good point, nevermind so :)